index.html

```diff
@@ -843,26 +843,26 @@ <h2 id="4Concepts.Sourceimage">Images</h2>
     <section id="4Concepts.ColourSpaces">
       <h2>Colour spaces</h2>
```

Comment:
Seems like this should be renamed. (Not the ID of course.)

Reply:
Good catch! I see that I had "match case" on in my search-and-replace, so I just did another pass changing Colour to Color as needed.